JASP-Desktop/components/JASP/Widgets/FileMenu/PrefsAdvanced.qml

```diff
@@ -206,10 +206,45 @@ ScrollView
 				checked:			preferencesModel.generateMarkdown
 				onCheckedChanged:	preferencesModel.generateMarkdown = checked
 				toolTip:			qsTr("Enabling this will generate markdown helpfile from the info at qml options.")
+				KeyNavigation.tab:	checkForLC_CTYPE_C
+				KeyNavigation.down:	checkForLC_CTYPE_C
+
+			}
+		}
+		
+		PrefsGroupRect
+		{
+			id:			windowsSpecific
+			visible:	WINDOWS
+			enabled:	WINDOWS
+			title:		qsTr("Windows Workarounds")
```

Comment:
`qsTr("Windows workarounds")`

Reply:
Okidoki, I just have a tendency to Capitalize a lot ;P